projects/vdk-plugins/vdk-impala/tests/functional/template_regression_test.py

```diff
@@ -108,6 +109,56 @@ def test_load_dimension_scd1_bad_target_schema(self) -> None:
             template_args=template_args,
         )
 
+    def test_load_dimension_scd1_checks_positive(self) -> None:
+
+        test_schema = "vdkprototypes"
+        source_view = "vw_dim_org"
+        target_table = "dw_dim_org"
```

Comment:
To avoid situaitons in which another test populates the data in a table. Let's use distinct names

```
source_view="vw_dim_org_check_positive"
target_table ="dw_dim_org_check_positive"

```

Reply:
Done.